CONTRIBUTING.org

```diff
@@ -20,11 +20,23 @@ make plain
 #+end_src
 
 * Contributing code
-The basic code style guide is to use:
+Here are the important settings:
 #+begin_src elisp
 (setq indent-tabs-mode nil)
+(setq lisp-indent-function #'common-lisp-indent-function)
 #+end_src
-It is provided for you in [[https://github.com/abo-abo/swiper/blob/master/.dir-locals.el][.dir-locals.el]], please obey it.
+
+The setting for =indent-tabs-mode= is automatically applied by
+=.dir-locals.el=. In Emacs 26, you can apply the setting for
+=lisp-indent-function= without changing it globally by adding
+
+#+begin_src elisp
+(add-to-list 'safe-local-variable-values
+             '(lisp-indent-function . common-lisp-indent-function))
+#+end_src
+
+to your init-file, copying =.dir-locals.el= to =.dir-locals-2.el=, and
+uncommenting the line in that file which sets =lisp-indent-function=.
```

Comment:
If we really are going to go full `common-lisp-indent-function` (on which decision I've commented in #1601), wouldn't it make more sense to uncomment the correspoding `dir-locals-file` line?

Reply:
By default, Emacs complains about the unsafe file-local variable, every time you open any file in the repository. That seems overly annoying, although my mind could be changed here.

Reply:
> By default, Emacs complains about the unsafe file-local variable, every time you open any file in the repository. That seems overly annoying

Indeed, which is why I thought we were suggesting the `safe-local-variable-values` spiel. I realise now, though, that "normal" users (i.e. users not hacking on Swiper) deserve to not have to play with `safe-local-variable-values` in order to quietly read the Swiper sources.